engine/table/src/main/java/io/deephaven/engine/table/impl/QueryTable.java

```diff
@@ -982,6 +980,37 @@ public static class FilteredTable extends QueryTable implements WhereFilter.Reco
         private boolean refilterUnmatchedRequested = false;
         private MergedListener whereListener;
 
+        @ReferentialIntegrity
+        private Runnable delayedErrorReference;
+
+        private static final class DelayedErrorNotifier implements Runnable {
```

Comment:
You should rebase and use the version of this that dropped in `io.deephaven.engine.table.impl.util`.

See https://github.com/deephaven/deephaven-core/blob/main/engine/table/src/main/java/io/deephaven/engine/table/impl/util/DelayedErrorNotifier.java.